### PR TITLE
Add disposable creation intro

### PR DESCRIPTION
**Pull request by gordon-shumway-net**
As also discussed on the QubesOS Summit 2023 with @marmarta , here a short intro regarding named disposables and disposable templates.
I hope the writing applies so far to the documentation style guide.

### CODE REVIEW

File: user/how-to-guides/how-to-use-disposables.md [hunk 1]
```diff
@@ -22,6 +22,22 @@ From inside an app qube, choosing the `Open in disposable` option on a file will
 
 This diagram provides a general example of how disposables can be used to safely open untrusted links and attachments in disposables. See [this article](https://blog.invisiblethings.org/2010/06/01/disposable-vms.html) for more on why one would want to use a disposable.
 
+## Named disposables and disposable templates
+
+There is a difference between [named disposables](/doc/glossary/#named-disposable) and [disposable templates](/doc/glossary/#disposable-template).
+
+In a default QubesOS Installation, you would probably use the 'whonix-ws-16-dvm' if you, as an example, want to browse the Tor network with an disposable. Every application starts a new random disposable with an ID in the name and if you close the window, it shuts down the qube. This is the feeling of an disposable template.
+
+In named disposables every application starts in the same qube, the qube itself has a fixed name and you need to manually shutdown the qube. Except from the non-persistance, they feel like usual app qubes. Named disposables are built upon disposable templates.
```
Comment (marmarta) [inline user/how-to-guides/how-to-use-disposables.md:31]:
```suggestion
Named disposable qubes function a bit differently - every application starts in the same qube, the qube itself has a fixed name and you need to manually shut it down. The only difference between a named disposable qube and a usual app qube is that a named disposable qube is not persistent - any changes are wiped when the qube is shut down. Named disposable qubes are, like normal disposable qubes, built upon disposable templates.
```

File: user/how-to-guides/how-to-use-disposables.md [hunk 2]
```diff
@@ -22,6 +22,22 @@ From inside an app qube, choosing the `Open in disposable` option on a file will
...
+
+### How to create disposable templates
+
+First you need to create an app qube. After that you need to go to the 'Qubes Settings' of the created app qube and set it as a 'Disposable template' in the 'Advanced' section and apply the change. From now on the entry in the Application menu is not named 'Qube' anymore, but splitted into 'Disposable' and 'Template (disp)'. The settings for the disposable can be changed under **'Application Menu -> Template (disp) -> Template: Qubes Settings**
```
Comment (marmarta) [inline user/how-to-guides/how-to-use-disposables.md:35]:
```suggestion
First, you need to create an app qube. You can run it normally, set up any necessary settings (like browser settings) you wish to be applied to every disposable qube ran from this template. Next, go to 'Qube Settings' of the app qube, set it as a _Disposable template_ in the _Advanced_ section and apply the change. 

In Qubes 4.1, from now on, the entry in the Application menu is not named 'Qube' anymore, but split into 'Disposable' and 'Template (disp)'. The settings for the disposable can be changed under **'Application Menu -> Template (disp) -> Template: Qubes Settings**

In Qubes 4.2, the qube will now appear in the menu as a disposable template (in the Apps section), from which you can launch new disposable qubes. To change the settings of the template itself or run programs in it, use the menu position for this qube located in the Templates section.
```

File: user/how-to-guides/how-to-use-disposables.md [hunk 3]
```diff
@@ -22,6 +22,22 @@ From inside an app qube, choosing the `Open in disposable` option on a file will
...
+
+### How to create named disposables
+
+Named disposables can be created under **Application Menu -> Create Qubes VM**, the type needs to be 'DisposableVM'.
```
Comment (marmarta) [inline user/how-to-guides/how-to-use-disposables.md:39]:
```suggestion
In Qubes 4.1: named disposables can be created under **Application Menu -> Create Qubes VM**, set the qube type  to be _DisposableVM_.

In Qubes 4.2: named disposables can be created by **Application Menu -> Settings -> Qubes Settings -> Create New Qube**. Set the qube type to _Named disposable_
```

**Review by marmarta — CHANGES_REQUESTED** [review]
Some minor rewording + I added info on qubes 4.2 where it was relevant. Great job!

File: user/how-to-guides/how-to-use-disposables.md (hunk 1 above)
Comment (gordon-shumway-net) [reply user/how-to-guides/how-to-use-disposables.md:31]: Is "any changes are wiped when the qube is shut down" really the best wording?
I thought the changes are just gone but not explicitly overwritten.

**Review by gordon-shumway-net — COMMENTED** [review]
So far so good, commited all suggested changes and left one small sentence, regarding ephermal states and if they are overwritten, open for discussion.

File: user/how-to-guides/how-to-use-disposables.md (hunk 1 above)
Comment (marmarta) [reply user/how-to-guides/how-to-use-disposables.md:31]: Yeah, this is a good point. Howeber, I think more explicitly describing it is a good idea? Maybe more like "changes are discarded"?

**Review by marmarta — CHANGES_REQUESTED** [review]

**Comment by marmarta** [comment]
Sorry for late answer, got bogged down in stuff!

**Pull request merged**